index.html

```diff
@@ -4006,11 +4006,9 @@ <h2><span class="chunk">mDCv</span>Mastering Display Color Volume</h2>
           </aside>
 
 
-          <aside class="note">
-            Below are mDCv examples for [[SMPTE ST 2086]] with SDR content that may be native or 
+          <p>Below are mDCv examples for [[SMPTE ST 2086]] with SDR content that may be native or 
```

Comment:
@ProgramMax was removing the "note" intended?

Reply:
It was. This didn't seem to be a note. It seemed more like "Here is a list of HDR examples. Here is a list of SDR examples."

Reply:
Ok. More than the "formatting" implied by the subject of the PR.

Reply:
You're right.

I typically do the 1-line commit title and then a multi-line commit message that describes in greater detail.
GitHub makes the multi-line message the first comment on the pull request.
The message says that it also alters things beyond formatting.

But I should have included that in the title somehow.